package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-data-schema",
-  "version": "16.0.0",
+  "version": "16.1.0",
```

Comment:
Can we bump major for this one since scmContext is required in a `get` for pipelines?